bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java

```diff
@@ -402,12 +411,25 @@ public synchronized BigtableTableAdminClient getTableAdminClient() throws IOExce
   }
 
   /**
-   * <p>Getter for the field <code>adminClientWrapper</code>.</p>
+   * Initializes bigtableTableAdminClient based on flag to use GCJ adapter, available in
+   * {@link BigtableOptions}.
    *
    * @return a {@link BigtableTableAdminClientWrapper} object.
    * @throws java.io.IOException if any.
    */
+  //TODO(rahulkql): rename the getter to getTableAdminClient.
   public synchronized IBigtableTableAdminClient getTableAdminClientWrapper() throws IOException {
+    if (options.useGCJClient()) {
+      if (adminGCJClient == null) {
+        //TODO(rahulkql): Decide If these clients need to be part of #close
+        com.google.cloud.bigtable.admin.v2.BigtableTableAdminClient adminClientV2 =
```

Comment:
can you import com.google.cloud.bigtable.admin.v2.BigtableTableAdminClient ?

Reply:
As there is already a BigtableTableAdminClient is being used in other methods, So had to use fully qualified names.